### PR TITLE
Cohort definitions were added to export/import

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes https://github.com/OHDSI/Atlas/issues/1821

### CODE REVIEW

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 1]
```diff
@@ -332,7 +333,25 @@ public IRAnalysisDTO createAnalysis(IRAnalysisDTO analysis) {
     return getAnalysis(createdAnalysis.getId());
   }
 
-  @Override
+    private void clearCohorts(IncidenceRateAnalysisDetails details) {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Cannot we encapsulate this in custom deserializer for IRAnalysisDTO?

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 2]
```diff
@@ -316,25 +324,110 @@ public IRAnalysisDTO createAnalysis(IRAnalysisDTO analysis) {
...
+          IncidenceRateAnalysis analysis = this.irAnalysisRepository.findOne(id);
+          ExceptionUtils.throwNotFoundExceptionIfNull(analysis, String.format(NO_INCIDENCE_RATE_ANALYSIS_MESSAGE, id));
+
+          try {
+              IncidenceRateAnalysisExpression expression = objectMapper.readValue(
+                      analysis.getDetails().getExpression(), IncidenceRateAnalysisExpression.class);
+
+              // Cohorts are not stored in expression now - create lists of cohorts from
+              // lists of their ids
+              fillCohorts(expression.outcomeIds, expression.outcomeCohorts);
```
Comment (anthonysena) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Question: do we need to retrieve the cohort expression for every cohort reference in the IR analysis? I think this is only necessary when we export the analysis unless I'm missing something in the approach designed here.

Comment (pavgra) [reply src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: @anthonysena , I agree with you - will update the PR

**Comment by pavgra** [comment]
Rebuild according to review in https://github.com/OHDSI/WebAPI/pull/1172

**Pull request closed without merging**